.github/workflows/test.yml

```diff
@@ -108,6 +152,17 @@ jobs:
       uses: actions/setup-python@v4
       with:
         python-version: 3.8
+    - name: Check for cache
+      uses: actions/cache@v3
+      with:
+        path: |
+          ${{ env.pythonLocation }}
+          ${{ env.LD_LIBRARY_PATH }}
+          ${{ github.workspace }}/.pytest_cache
```

Comment:
Does .pytest_cache help in any of the jobs, when pytest is invoked without -ff or -lf?

Reply:
yes it does, i'm not sure about the exact implementation but i believe it kinds of skips when the file tested is unchanged. `-ff` and `-lf` is for when you have failure cases only by only running on them.